perfkitbenchmarker/linux_benchmarks/container_netperf_benchmark.py

```diff
@@ -27,6 +27,9 @@
 
 FLAGS = flags.FLAGS
 
+flags.DEFINE_integer('container_netperf_tcp_stream_send_size_in_bytes', 131072,
```

Comment:
Please add a comment as to why this default value was selected.

Reply:
This value, 131072 bytes (128KB) was selected at the recommendation of @jonesrick. 

"The 128K send size is deliberate - otherwise netperf will default to 16K based on Linux send socket buffer size initial values, which has enough overhead to preclude achieving "link rate" as it were.  And the default receive size will be 87380 bytes for similar reasons on the receive side. That would be a good thing to change in the netperf command-lines in PKB.  And it would match the default send size in iperf3.  64K may be sufficient but 128K would give additional cushion.  The need for receive side (-M) isn't as clear-cut but it won't hurt." 

Or, if preferred, I could give it a default of None and let netperf select the size unless this flag is set.

Reply:
I think setting the default is important. I'd just like a comment in the code so that we know why we selected this default. Noting that the Linux default of 16K cannot achieve "link rate" and that the higher default value will let us would be sufficient.

Reply:
Do we also need to set `-s`?

Per https://hewlettpackard.github.io/netperf/doc/netperf.html#UDP_005fSTREAM
If the value of the -m option is larger than the local send socket buffer size (-s option) netperf will likely abort with an error message about how the send call failed:

     netperf -t UDP_STREAM -H 192.168.2.125
     UDP UNIDIRECTIONAL SEND TEST from 0.0.0.0 (0.0.0.0) port 0 AF_INET to 192.168.2.125 (192.168.2.125) port 0 AF_INET
     udp_send: data send error: Message too long

Reply:
Added a comment to clarify default size. Addressed -s flag in a comment below

Reply:
Thanks for adding the comment. Since comments are easier to parse if they contain punctuation and consist of complete sentences, perhaps edit it so that it reads more like the following:

# We set the default to 128KB (131072 bytes) to override the Linux default
# of 16K so that we can achieve the "link rate".

Reply:
Hmm. Looks like the prefix of "#" changed my font size. I just meant to write a comment.

Reply:
changed the comment